lib/ProductOpener/Config_off.pm

```diff
@@ -301,6 +303,19 @@ $options{product_type} = "food";
 		],
 	},
 
+	# as of 2023-01-27 far too much errors in updates
+	# No fix on the app
+	{
+		name => "Halal App Chakib",
+		conditions => [["user_id", "halal-app-chakib"],],
+		actions => [["ignore"],],
```

Comment:
So you removed the rules on the fields? They did not work as expected?

			["ignore_if_existing_categories"],
			["ignore_if_existing_categories_tags"],
			["ignore_if_existing_categories_en"],
			["ignore_if_existing_categories_tags_en"],
			["ignore_if_existing_countries"],
			["ignore_if_existing_countries_tags"],
			["ignore_if_existing_added_countries_tags"],
			["ignore_if_existing_countries_en"],
			["ignore_if_existing_countries_tags_en"],
			["ignore_if_existing_added_countries_tags_en"],

Reply:
Because I though I wouldn't catch edit on other languages, and was unsure if there where other fields impacted. That's why with @teolemon we decided to go for "ignore" (and just keep images).

Reply:
But your opinion matters :-)

Reply:
It's fine to ignore, I was just curious and wanted to know if the previous rules didn't work.